README.md

```
@@ -112,29 +112,30 @@ See [Frappe Docker](https://github.com/frappe/frappe_docker/blob/main/docs/01-ge
+### Option 1: Local Development (Recommended for Contributors)
 
-### Local
+This method is recommended if you plan to contribute to ERPNext.
 
-To setup the repository locally follow the steps mentioned below:
+1. Setup bench by following the official installation guide:  
+   https://frappeframework.com/docs/user/en/installation  
 
-1. Setup bench by following the [Installation Steps](https://frappeframework.com/docs/user/en/installation) and start the server
+2. Start the development server:
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)